lighthouse-cli/bin.ts

```diff
@@ -52,6 +52,12 @@ const cliFlags = yargs
   .showHelpOnFail(false, 'Specify --help for available options')
 
   .usage('$0 url')
+  .example('$0 url --view', 'Opens the HTML report in a browser after the run completes')
```

Comment:
IMO we should just use "lighthouse" rather than whatever $0 resolves to.

how about with brackets on the url too: `lighthouse <url> --view`